pkg/virtual/framework/dynamic/apiserver/serving_info.go

```diff
@@ -108,6 +109,19 @@ func CreateServingInfoFor(genericConfig genericapiserver.CompletedConfig, apiRes
 		return nil, err
 	}
 
+	openAPI, err := buildOpenAPIV3(
+		apiResourceSchema,
+		apiResourceVersion,
+		builder.Options{
+			V2: true,
+			SkipFilterSchemaForKubectlOpenAPIV2Validation: true,
+			StripValueValidation:                          true,
+			StripNullable:                                 true,
```

Comment:
is upstream also stripping value validation and nullable? Looks odd.

Reply:
I copied the ones in buildV2 to make minimal changes.

When grep'ed through upstream, I didn't see these to be modified at all. Will drop.